core/tests/test_utils.py

```diff
@@ -53,6 +53,9 @@
 
 
 def empty_environ():
+    """Set up empty environment variables for tests 
```

Comment:
In this file, we seem to be using a mix of third-person plural and third-person singular (e.g. "Set ..." vs "Sets ..."). Could we standardize for consistency? I think third-person singular is more common (e.g. "Sets ..."). 

Ditto below.